mapreduce/tests/test_mapreduce.py

```diff
@@ -24,6 +24,8 @@
     RM_URI,
 )
 
+pytestmark = pytest.mark.unit
```

Comment:
(nit) Not sure why this is needed.

Reply:
It's not _needed_  but lets you do `ddev test <integration>` -m unit.